canarytokens/channel_dns.py

```diff
@@ -118,6 +118,7 @@ def _do_ns_response(self, name=None):
             payload=dns.Record_A(ttl=10, address=self.frontend_settings.PUBLIC_IP),
             type=dns.A,
             auth=True,
+            ttl=10
```

Comment:
```suggestion
            ttl=300
```

Reply:
If we want to change NS TTLs, we should change it in the Record_A too?